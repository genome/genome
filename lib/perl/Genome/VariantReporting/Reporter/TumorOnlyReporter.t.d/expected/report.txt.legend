Headers
chromosome_name	Chromosome: an identifier from the reference genome or an angle-bracketed ID String ("<ID>") pointing to a contig in the assembly file
start	The start position of the variant. One-based
stop	The end position of the variant. One-based, inclusive
reference	The reference allele at this position
variant	The variant called at this position
rsid	A list of rsids at this position
transcript_name	Ensembl stable ID of feature
trv_type	Consequence type of this variation
amino_acid_change	The HGVS protein sequence name
default_gene_name	The gene symbol
gene_name_source	The source of the gene symbol
ensembl_gene_id	Ensembl stable ID of affected gene
c_position	The HGVS coding sequence name
canonical	Is this transcript canonical
sift	The SIFT prediction and/or score, with both given as prediction(score)
polyphen	The PolyPhen prediction and/or score
condel	The Consensus Deleteriousness (Condel) score
caf	Allele frequency at this position based on 1000Genomes
All_MAF	Overall minor allele frequency from NHLBI
AA_MAF	African American minor allele frequency from NHLBI
EU_MAF	European minor allele frequency from NHLBI
1kg-af	Allele frequency based on thousand genomes project
vaf	Variant allele frequency for sample sample_name
ref_count	Number of reads supporting the reference for sample sample_name
var_count	Number of reads supporting variant for sample sample_name
Filters
