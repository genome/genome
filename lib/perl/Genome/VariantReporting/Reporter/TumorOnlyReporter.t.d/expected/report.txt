chromosome_name	start	stop	reference	variant	rsid	transcript_name	trv_type	amino_acid_change	default_gene_name	gene_name_source	ensembl_gene_id	c_position	canonical	sift	polyphen	condel	caf	All_MAF	AA_MAF	EU_MAF	1kg-af	vaf	ref_count	var_count
1	1	1	A	T	rs1	ENST00000452176	DOWNSTREAM		RP5-857K21.5	HGNC	ENSG00000223659	c.456	1	deleterious(5.4)	-	-	.1	0.1	0.3	0.1	0.001	30	0	200
1	1	1	A	G	rs1	ENST00000452176	DOWNSTREAM		RP5-857K22.5	HGNC	ENSG00000223695	c.456	1	-	-	-	.1	0.1	0.3	0.1	0.001	70	0	1000
